ext/pgsql/php_pgsql.h

```diff
@@ -178,17 +178,17 @@ static const php_stream_ops php_stream_pgsql_fd_ops = {
 #define PGSQL_MAX_REGEXES 11
 
 ZEND_BEGIN_MODULE_GLOBALS(pgsql)
+	HashTable field_oids;
+	HashTable table_oids;
+	HashTable connections;
 	zend_long num_links,num_persistent;
 	zend_long max_links,max_persistent;
```

Comment:
If I compute this correctly, you are splitting the max_links and max_persistent fields on different cache lines. This might be worse than having a tiny bit of padding between the bools and the 3 HTs at the end.

Reply:
yes that s true ... let me see.